### PR TITLE
ec_deployment: Add `drop_pipeline` variable

### PR DESCRIPTION
**Pull request by marclop**
## Motivation/summary

Introduces a new `drop_pipeline` variable which can be used to measure the theoretical maximum output from the APM Server.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

## How to test these changes

Enable `drop_pipeline` in `testing/benchmark`.

## Related issues

Seemed from #9180

### CODE REVIEW

**Review by axw — COMMENTED** [review]
Looks good, but I think the script is missing. I'm curious to know how the pipeline is used. Modifying the custom component templates? Custom ingest pipelines?

**Comment by marclop** [comment]
@axw indeed! Sorry to keep you waiting and on the dark. I've added the script in 411fbe9

File: testing/infra/terraform/modules/ec_deployment/scripts/drop_pipeline.tftpl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eo
+
+ingest_pipelines=("logs-apm.app@custom" "logs-apm.error@custom" "metrics-apm.app@custom" "metrics-apm.internal@custom" "traces-apm.rum@custom" "traces-apm@custom")
```
Comment (axw) [inline testing/infra/terraform/modules/ec_deployment/scripts/drop_pipeline.tftpl:5]: This is fine for now, but may be a little brittle and friction-inducing as we introduce more data streams. I've had this thought in my mind for a little while, and finally have written an issue to implement it: https://github.com/elastic/apm-server/issues/9339. If/when we do, we should come back and update this to make use of it.

**Review by axw — APPROVED** [review]
LGTM!

**Pull request merged**